library/akamai.py

```diff
@@ -1,29 +1,130 @@
-#!/usr/bin/env python
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""
+Provides authentication using local `.edgerc` files for Akamai API calls.
+Note this replaces the ansible.builtin.uri module which will not do
+Akamai authentication with `.edgerc` files.
+"""
+
+# Copyright: (c) 2018, Jacob Hudson <jacob.alan.hudson@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
-from ansible.module_utils.basic import *
-import requests
 import json
 
+# import urllib.parse
```

Comment:
If urllib isn't needed can this line be removed?

Reply:
Yep, left in error. Thanks!